ui/components/institutional/compliance-settings/compliance-settings.js

```diff
@@ -0,0 +1,98 @@
+import React, { useContext } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  JustifyContent,
+  DISPLAY,
+  TextColor,
+  FLEX_DIRECTION,
+} from '../../../helpers/constants/design-system';
+import { I18nContext } from '../../../contexts/i18n';
+import { mmiActionsFactory } from '../../../store/institutional/institution-background';
+import { Text } from '../../component-library';
+import Box from '../../ui/box';
+import Button from '../../ui/button';
+
+const ComplianceSettings = () => {
+  const t = useContext(I18nContext);
+  const dispatch = useDispatch();
+  const mmiActions = mmiActionsFactory();
+
+  const complianceActivated = useSelector((state) =>
+    Boolean(state.metamask.institutionalFeatures?.complianceProjectId),
+  );
+
+  const disconnectFromCompliance = async () => {
+    await dispatch(mmiActions.deleteComplianceAuthData());
+  };
+
+  const renderDisconnect = () => {
+    return (
+      <Button
+        type="default"
+        large
+        onClick={disconnectFromCompliance}
+        data-testid="disconnect-compliance"
+      >
+        {t('disconnect')}
+      </Button>
+    );
+  };
+
+  const renderLinkButton = () => {
+    return (
+      <Button
+        type="primary"
+        data-testid="start-compliance"
+        onClick={() => {
+          global.platform.openTab({
+            url: 'https://start.compliance.codefi.network/',
+          });
+        }}
+      >
+        {t('openCodefiCompliance')}
+      </Button>
+    );
+  };
```

Comment:
As this is reused it may not fall under the same rule as above regarding declaration. However, it doesn't need to be a method. Just assign the JSX to the const and render it.